pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -833,6 +833,11 @@ func (m *ManagerImpl) allocateContainerResources(pod *v1.Pod, container *v1.Cont
 		resource := string(k)
 		needed := int(v.Value())
 		klog.V(3).InfoS("Looking for needed resources", "resourceName", resource, "pod", klog.KObj(pod), "containerName", container.Name, "needed", needed)
```

Comment:
I would probably have moved the guard before this log so we log once per resource vs twice, but this is so minor I won't bother resubmit